x/ibc/evm/client/cli/query.go

```diff
@@ -0,0 +1,60 @@
+package cli
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/spf13/cobra"
+
+	"github.com/evmos/evmos/v10/x/ibc/evm/types"
+)
+
+// GetQueryCmd returns the cli query commands for this module
+func GetQueryCmd() *cobra.Command {
+	// Group epochs queries under a subcommand
+	cmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      fmt.Sprintf("Querying commands for the %s module", types.ModuleName),
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	cmd.AddCommand(
+		GetParamsCmd(),
+	)
+
+	return cmd
+}
+
+// GetParamsCmd queries ibc-evm-tx module params
+func GetParamsCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "params",
+		Short: "Gets ibc-evm-tx params",
+		Long:  "Gets ibc-evm-tx params",

```

Comment:
We renamed the module to IBC EVM to be more generic. So in the future we could do queries also for example
```suggestion
// GetParamsCmd queries ibc-evm module params
func GetParamsCmd() *cobra.Command {
	cmd := &cobra.Command{
		Use:   "params",
		Short: "Gets ibc-evm params",
		Long:  "Gets ibc-evm params",

```

Reply:
could also use 

```
fmt.Sprintf("Gets %s  params", types.ModuleName)
```